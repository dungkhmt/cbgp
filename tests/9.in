50 50
50 100
32 49
43 39
46 24
24 48
43 34
45 8
34 4
33 44
6 0
28 4
36 7
36 13
17 5
46 11
36 9
42 6
15 8
20 1
4 5
4 6
8 5
18 17
35 11
12 17
47 11
38 20
45 14
49 4
41 46
30 21
6 2
31 1
22 16
20 27
8 9
23 17
31 12
29 24
13 26
48 39
29 3
25 19
22 2
30 19
12 44
45 10
40 16
31 21
37 12
1 2
45 3
14 3
37 0
49 2
42 22
35 14
15 9
33 17
5 34
42 16
13 1
8 1
14 11
42 27
19 0
45 7
6 1
29 11
30 25
23 37
39 32
32 4
21 19
18 23
17 0
31 19
31 0
46 29
2 3
23 12
26 7
43 32
48 11
13 9
42 20
26 3
35 47
9 7
41 48
45 9
33 31
39 2
33 8
33 12
13 7
49 39
38 42
23 0
46 48
32 34
1
48 8 10 5 34 22 38 26 62 16 57 0 42 14 36 28 2 0 33 31 0 0 48 30 15 2 32 27 43 31 31 28 37 18 55 1 51 3 43 12 28 15 45 9 34 19 51 4 48 26 42 10 37 27 33 15 76 1 44 26 44 10 12 4 55 26 9 1 82 0 44 32 34 29 50 5 31 18 53 25 35 20 49 27 32 17 55 27 12 3 37 37 48 29 45 33 52 26 54 24 
