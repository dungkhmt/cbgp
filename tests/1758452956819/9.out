(32, 49), (43, 39), (46, 24), (24, 48), (43, 34), (45, 8), (34, 4), (33, 44), (6, 0), (28, 4), (36, 7), (36, 13), (17, 5), (46, 11), (36, 9), (42, 6), (15, 8), (20, 1), (4, 5), (4, 6), (8, 5), (18, 17), (35, 11), (12, 17), (47, 11), (38, 20), (45, 14), (49, 4), (41, 46), (30, 21), (6, 2), (31, 1), (22, 16), (20, 27), (8, 9), (23, 17), (31, 12), (29, 24), (13, 26), (48, 39), (29, 3), (25, 19), (22, 2), (30, 19), (12, 44), (45, 10), (40, 16), (31, 21), (37, 12), (1, 2), (45, 3), (14, 3), (37, 0), (49, 2), (42, 22), (35, 14), (15, 9), (33, 17), (5, 34), (42, 16), (13, 1), (8, 1), (14, 11), (42, 27), (19, 0), (45, 7), (6, 1), (29, 11), (30, 25), (23, 37), (39, 32), (32, 4), (21, 19), (18, 23), (17, 0), (31, 19), (31, 0), (46, 29), (2, 3), (23, 12), (26, 7), (43, 32), (48, 11), (13, 9), (42, 20), (26, 3), (35, 47), (9, 7), (41, 48), (45, 9), (33, 31), (39, 2), (33, 8), (33, 12), (13, 7), (49, 39), (38, 42), (23, 0), (46, 48), (32, 34), 
numberIntersection=0 
minDistance=2.0 
sumDistance=579806.9812685859 
minAngle=0.011493746715030895 
sumAngle=30.67245298612012 
minEdgeLength=2.0 
sumEdgeLength=4665.425111846709 

0.0000000000 2.0000000000 579806.9812685859 0.0114937467 30.6724529861 2.0000000000 4665.4251118467
0: (180, 13),
1: (5, 9),
2: (11, 90),
3: (13, 176),
4: (181, 15),
5: (193, 0),
6: (91, 25),
7: (5, 164),
8: (2, 0),
9: (4, 146),
10: (0, 1),
11: (5, 188),
12: (18, 4),
13: (10, 146),
14: (0, 179),
15: (6, 126),
16: (11, 18),
17: (188, 2),
18: (181, 5),
19: (68, 18),
20: (8, 12),
21: (70, 12),
22: (8, 18),
23: (181, 8),
24: (58, 193),
25: (64, 14),
26: (7, 167),
27: (12, 13),
28: (194, 2),
29: (4, 181),
30: (67, 14),
31: (11, 8),
32: (107, 198),
33: (7, 2),
34: (200, 0),
35: (0, 194),
36: (7, 148),
37: (178, 10),
38: (8, 15),
39: (105, 200),
40: (11, 20),
41: (53, 195),
42: (11, 15),
43: (132, 200),
44: (12, 5),
45: (2, 175),
46: (47, 193),
47: (0, 200),
48: (56, 200),
49: (105, 195),
180 13 5 9 11 90 13 176 181 15 193 0 91 25 5 164 2 0 4 146 0 1 5 188 18 4 10 146 0 179 6 126 11 18 188 2 181 5 68 18 8 12 70 12 8 18 181 8 58 193 64 14 7 167 12 13 194 2 4 181 67 14 11 8 107 198 7 2 200 0 0 194 7 148 178 10 8 15 105 200 11 20 53 195 11 15 132 200 12 5 2 175 47 193 0 200 56 200 105 195 
0: (180, 13), 1: (5, 9), 2: (11, 90), 3: (13, 176), 4: (181, 15), 5: (193, 0), 6: (91, 25), 7: (5, 164), 8: (2, 0), 9: (4, 146), 10: (0, 1), 11: (5, 188), 12: (18, 4), 13: (10, 146), 14: (0, 179), 15: (6, 126), 16: (11, 18), 17: (188, 2), 18: (181, 5), 19: (68, 18), 20: (8, 12), 21: (70, 12), 22: (8, 18), 23: (181, 8), 24: (58, 193), 25: (64, 14), 26: (7, 167), 27: (12, 13), 28: (194, 2), 29: (4, 181), 30: (67, 14), 31: (11, 8), 32: (107, 198), 33: (7, 2), 34: (200, 0), 35: (0, 194), 36: (7, 148), 37: (178, 10), 38: (8, 15), 39: (105, 200), 40: (11, 20), 41: (53, 195), 42: (11, 15), 43: (132, 200), 44: (12, 5), 45: (2, 175), 46: (47, 193), 47: (0, 200), 48: (56, 200), 49: (105, 195), 
numberIntersection=0 minAngle=0.011493746715030895 sumAngle=30.67245298613204 minEdgeLength=2.0 sumEdgeLength=4665.425111846687 minDistance=2.0 sumDistance=579806.9812685859 
Time taken: 17.046 s
